### PR TITLE
arm pb definitely due to poor denormals handling

### PR DESCRIPTION
**Pull request by jtlap**
also putting pedantic hypot to 1ulp

### CODE REVIEW

File: test/unit/module/math/hypot.cpp [hunk 1]
```diff
@@ -72,6 +72,8 @@ TTS_CASE_TPL("Check return types of hypot", eve::test::simd::ieee_reals)
   TTS_EXPR_IS(pedantic(hypot)(v_t(), int(), T()), T);
   TTS_EXPR_IS(pedantic(hypot)(v_t(), T(), int()), T);
   TTS_EXPR_IS(pedantic(hypot)(v_t(), v_t(), v_t()), v_t);
+  std::cout << hypot(0.0014, 1.1e+02) << std::endl;
```
Comment (jfalcou) [inline test/unit/module/math/hypot.cpp:75]: remove

File: test/unit/module/math/hypot.cpp [hunk 2]
```diff
@@ -116,28 +118,31 @@ TTS_CASE_WITH("Check corner-cases behavior of eve::hypot variants on wide",
 <typename T>(T const& a0)
 {
   auto cases = tts::limits(tts::type<T> {});
-
+#ifdef SPY_ARCH_IS_ARM
```
Comment (jfalcou) [inline test/unit/module/math/hypot.cpp:121]: why do we hide more tests than before ?

**Review by jfalcou — APPROVED** [review]
